### ISSUE TITLE
Category Filter on Home

### ISSUE THREAD

**Issue by ilanza24** [chingu-voyages/v47-tier2-team-13#20]
**User Story:** 
As user, I want filter my tasks based on their categories, so I can structure my schedule based on the type

**Acceptance Criteria:**
- [ ] filter is visible and user is able to select from a set of categories
- [ ] tasks are showing based on selected category

**Comment by ilanza24** [comment]
@Zien-Alhawshi @AlexShefer this is the filter system that shows in the daily tasks, not sure who needs to work on this. Feel free to change the assignment if you need

**Comment by Zien-Alhawshi** [comment]
Yep, I can handle this one, I really love working with Alex on it 
@AlexShefer the main functionality is:
the user see the current categories: all categories first, and when he/she click on a specific category, the tasks are filtered and the category's tasks are shown. 

This is right?

**Comment by ilanza24** [comment]
Yes, exactly, as shown in the prototype also
